### PR TITLE
Last changes for opends 0.3.0

### PR DESCRIPTION
**Pull request by samleeflang**
- Instituion -> Organisation change 
- Change in Agent in EntityRelationship
- Added status to digital specimen and digital media

### CODE REVIEW

**Review by TomDijkema — APPROVED** [review]
:shaved_ice:

File: src/test/java/eu/dissco/core/translator/TestUtils.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static String loadResourceFile(String fileName) throws IOException {
   public static DigitalSpecimen givenDigitalSpecimen() {
     return new DigitalSpecimen()
         .withOdsNormalisedPhysicalSpecimenID(NORMALISED_PHYSICAL_SPECIMEN_ID)
-        .withDwcInstitutionID(INSTITUTION_ID);
+        .withOdsOrganisationID(INSTITUTION_ID);
```
Comment (southeo) [inline src/test/java/eu/dissco/core/translator/TestUtils.java:74]: Maybe change the name of this constant?

**Review by southeo — DISMISSED** [review]
:postal_horn:

File: src/test/java/eu/dissco/core/translator/TestUtils.java (hunk 1 above)
Comment (samleeflang) [reply src/test/java/eu/dissco/core/translator/TestUtils.java]: Yes thanks done

**Pull request merged**